test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -744,6 +748,26 @@
              (is (= 200 (:status @response)))
              (is (= hello-body (:body @response))))))))))
 
+(deftest double-stop-test
+  (testing "if the stop lifecycle is called more than once, we handle that gracefully and quietly"
+    (let [log-messages (atom [])]
+      (tk-log-testutils/with-logging-to-atom tk-log/root-logger-name log-messages
+        (let [app (tk-bootstrap/bootstrap-services-with-config
+                   [jetty9-service]
+                   {:webserver {:port 8080}})]
+          (tk-app/stop app)
```

Comment:
Ah, k.  I didn't realize Jetty was catching the exception.  I'd been thinking Trapperkeeper was.  Yeah, I agree that testing for log messages might be the best we can do in this case, then.